python-regression/tests/features/machine6/6_local_snapshots_tests.feature

```diff
@@ -0,0 +1,93 @@
+Feature: Test Bootstrapping With LS
+  A test to determine whether or not nodes can bootstrap and sync correctly from Local Snapshot Files and DB's. One
+  permanode will be started containing all the relevant files/folders for a full sync upon start. Two more nodes will
+  be started, connected to this node and one another: One will have only a DB and snapshot file, while the other will
+  have only the snapshot meta and state file, along with the spent addresses DB and the snapshot file. All three nodes
+  should sync with one another. And a snapshot should be taken on the node started with just a DB.
+  [NodeA: Permanode, NodeB: Just DB, NodeC: Just LS Files]
+
+  Scenario: PermaNode is synced
+    Check that the permanode has been started correctly and is synced.
+
+    #First make sure nodes are neighbored
+    Given "nodeA" and "nodeB" are neighbors
+    And "nodeA" and "nodeC" are neighbors
+
+    # Wait to ensure node has time to sync
+    When we wait "10" second/seconds
```

Comment:
nit:
I remeber we said you will do a regex that will accept both "second" and "seconds"

Not very important though

Reply:
By the way I don't understand the scenario. Isn't node A always synced and the others need to get synced from it?
Why do we wait here?

Reply:
This scenario is to make sure that when the `db`, `snapshot.txt`, `spent-addresses-db`, and snapshot files the node does start correctly and is synced correctly.